pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -368,6 +368,8 @@ type KubeletConfiguration struct {
 	// runtimeRequestTimeout is the timeout for all runtime requests except long running
 	// requests - pull, logs, exec and attach.
 	RuntimeRequestTimeout unversioned.Duration `json:"runtimeRequestTimeout"`
+	// If no pulling progress is made before the deadline imagePullProgressDeadline, the image pulling will be cancelled.
```

Comment:
Updated the patch to add "Defaults to 1m0s.". It seems no need to add "this cannot be turned off" because the default value has tell this information.